src/PageTypes/BasePage.php

```diff
@@ -95,12 +95,8 @@ public function getFooter()
         return FooterHolder::get_one(FooterHolder::class);
     }
 
-    /**
-     * @deprecated 2.2.0 Use RelatedPagesThrough() instead
```

Comment:
Undeprecating because it's called a lot by templates in the starter theme.  Given lots of project will have copied the files into their own theme, I think forcing templates to update this to `<% loop $RelatedPagesThrough %>` (which may not even work) is adding too much upgrade friction